src/components/Molecules/Box/Box.js

```diff
@@ -27,10 +27,16 @@ const Copy = styled.div`
   padding: ${spacing('l')};
   display: flex;
   flex-direction: column;
-  justify-content: center;
   flex-grow: 0;
   flex-shrink: 0;
   flex-basis: calc(200% / 3);
```

Comment:
Can we refactor to flex: 0 0 calc(200% / 3);

Reply:
It is best practice to not use shorthand. IE can interpret different than other browsers and it is also better to read and understand.